components/elm/cime_config/config_compsets.xml

```diff
@@ -729,6 +729,11 @@
     <lname>1850_DATM%CPLWCH_ELM%CNPRDCTCBCWFM-CROP_SICE_SOCN_MOSART_SGLC_SWAV</lname>
   </compset>
 
+  <compset>
+   <alias>I20TRWCCNPWFMCROP</alias>
```

Comment:
A compset with the same name and description (I20TRWCCNPWFMCROP) exists on the lines immediately following the current addition. Please delete this new compset.

Reply:
@jinyuntang Yes, that would be my suggestion. @rljacob do you agree?

Reply:
You should definitely not add the same compset twice.  If you want to change the phys option (the text after the percent sign) to some other string (without changing what it does) then edit the existing compset alias.

Reply:
I think the dash helps with readability so would keep that version.  You can probably just close this PR.

Reply:
@rljacob  Thanks for clarifying. This PR does add additional compsets that do not exist in the master and should keep those new compsets but remove the ones that already exist.

Reply:
I'd like to move this discussion  to the main comment thread.

Reply:
oh I see.  There's more compsets.  So yes just decided how to resolve the conflict with the duplicate compsets.

Reply:
@rljacob and @evasinha OK. I will keep the ones with the dash. One concern though, if in the future, we have another new feature, should this mean we add another dash? Or perhaps, an underscore will be better?

Reply:
That's a good point.  If CROP is the only thing with a dash, then why not remove the dash so it looks like the other options?

In general, the land group needs to clean these up.  "ELM" with no options should build your default config that you always want to use.  Then just add "%" options to depart from that default.

Reply:
@evasinha, in the sense of minimizing confusion, I'd suggest we remove the "dash". What do you think?

Reply:
@jinyuntang Sounds good to me.